src/py/_pyodide/_core_docs.py

```diff
@@ -1226,7 +1226,7 @@ def to_js(
         result of the dict conversion. Some suggested values for this argument:
 
           * ``js.Map.new`` -- similar to the default behavior
-          * ``js.Array.from`` -- convert to an array of entries
+          * ``js.Array.new`` -- convert to an array of entries
```

Comment:
we want `Array.from_` here. `Array.new([1,2,3])` returns `[[1,2,3]]`, whereas `Array.from` returns `[1,2,3]`. You can access `Array.from` as `from_` in Python.

Reply:
Yes I played around with the behavior and you are right the result is a nested Array in Javascript